packages/components/src/utils/validation.ts

```diff
@@ -21,6 +21,28 @@ export const REF_HREF_PATTERN =
 export const REF_INTERNAL_HREF_PATTERN =
   `(${ALLOWED_URL_REGEXES.internal})|(${ALLOWED_URL_REGEXES.legacy})` as const
 
+// Validation for form-related embed URLs
+export const isValidFormSGEmbedUrl = (url: string) => {
+  if (!url) {
+    return false
+  }
+
+  try {
+    const urlObject = new URL(url)
+    return urlObject.hostname === "form.gov.sg"
```

Comment:
nit: im not sure if there are other valid formsg url that aren't actually forms, so might wanna check for the slug

Reply:
Yeah I'm not too sure also but was thinking of doing the same treatment here as with Maps.gov.sg to do no validation. Worst case is people embed weird stuff but it is still scoped to form.gov.sg so I think it's ok